### DISCUSSION TITLE
Nullability and Directory.Build.props

### DISCUSSION THREAD

**Discussion by Morbidous** [MNES-Emulator/MNES#20] [Q&A] [answered]
For some reason the main project has nullable disable. I really want to turn nullability on in that project.

Also I will create a `Directory.Build.props` file to have centralized properties (like Nullable enable).

Related to this, do you want the extra blank lines in your .csproj files? I would not usually have these blank lines but not sure if you want them or not. I will assume you don't care for now but we can add them back if you want them.

**Comment by Morbidous** [comment]
Also adding `Directory.Packages.props` which lets you define your package versions once in a central place.

**Comment by jamieyello** [comment, marked as answer]
I disabled it because I am lazy and don't like looking at all the squiggly lines, but I guess I can't hide from it forever. Yeah we should turn it back on.
